vignettes/rvar.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "The Random Variable Datatype"
+title: "rvar: The Random Variable Datatype"
```

Comment:
This is the only other change. pkgdown complained that the title is not equal to VignetteIndexEntry which is:

`%\VignetteIndexEntry{rvar: The Random Variable Datatype}`

I decided to add rvar: to the title, but I am not sure what the intention was here.

cc: @mjskay

Reply:
Fine with me. I did make those different deliberately for a reason I can't currently recall, but if pkgdown is gonna complain making them agree is fine with me

Reply:
We can always ignore that warning or turn it off when building. I didnt know it was intentional.

Reply:
Nah, it's less complicated to make them match. I think I was copying a style from somewhere else, but I can't find it again and I can't recreate the rationale I had to make them not match.